packages/reports/addon/templates/components/navi-request-column-config/base.hbs

```diff
@@ -0,0 +1,22 @@
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="navi-request-column-config" ...attributes>
+  <div class="navi-request-column-config__header">
+    <span class="navi-request-column-config__header-label">Column Configuration</span>
+    <span class="navi-request-column-config__header-icons">
+      <NaviIcon @icon="star-o" class="navi-request-column-config__star-icon" role="button"/>
```

Comment:
what does this do again?

Reply:
We'll be using it to favorite configurations for a column so that users can easily load a favorite config for a new or existing column. It's merely a placeholder in this PR.